jest-preset.js

```diff
@@ -12,5 +12,8 @@ module.exports = {
         plugins: ['@babel/plugin-transform-runtime']
       }
     ]
-  }
+  },
+  transformIgnorePatterns: [
+    'node_modules/(?!(nuxt-i18n)/)'
```

Comment:
nuxt-i18n is not the only module exporting esm syntax....

Reply:
No doubt, do we need to make a list of the modules, or do I know another way?

Reply:
Actually, I'm thinking to transpile `node_modules` (with auto esm detect system similar to jiti). There is a WIP implementation of esbuild transform that will make public soon :)

Reply:
@pi0  Thinking about using
https://www.npmjs.com/package/@swc/jest
or
https://github.com/aelbore/esbuild-jest
?
Any indications?

Reply:
I tried both. swc was so old and esbuild-jest is actually using babel!! (source: https://github.com/aelbore/esbuild-jest/blob/master/src/transformer.ts)

Publishing npm package this afternoon. it is inlined in nuxt3 repo.

Reply:
We could have [`jiti/jest`](https://github.com/unjs/jiti), no?
So it would be under our supervision.

Reply:
With this configuration its works, but increased runtime a little

```js
module.exports = {
  collectCoverage: true,
  testEnvironment: 'node',
  transformIgnorePatterns: [],
  transform: {
    '^.+\\.(t|j)sx?$': [
      'esbuild-jest'
    ]
  }
}
```

another approach would be to ignore only the modules used in nuxt.config

Reply:
I don't recommend transpiling all node_modules, often it's just one module or another, it's better to put a warning in the documentation

Reply:
Well slowly more and more npm packages are going to use `type: module` so we need to be also futuristic. But also we can use [jiti's approach](https://github.com/unjs/jiti/blob/57f0fbf05d27f98d6c369354b3f47d29f5732782/src/jiti.ts#L175) to smartly transpile when needed :)